clusterloader2/testing/nichotr/config.yaml

```diff
@@ -0,0 +1,143 @@
+#Constants
+{{$NODE_MODE := DefaultParam .NODE_MODE "allnodes"}}
+{{$NODES_PER_NAMESPACE := DefaultParam .NODES_PER_NAMESPACE 1}}
+{{$PODS_PER_NODE := DefaultParam .PODS_PER_NODE 56}}
+{{$ENABLE_CHAOSMONKEY := false}}
+
+#Pod Resource Request
+{{$POD_CPU := DefaultParam .POD_CPU 20}}
+{{$POD_MEMORY := DefaultParam .POD_MEMORY 100}}
+
+#Variables
+{{$namespaces := DivideInt .Nodes $NODES_PER_NAMESPACE}}
+{{$podsPerNamespace := MultiplyInt $PODS_PER_NODE $NODES_PER_NAMESPACE}}
+{{$totalPods := MultiplyInt $podsPerNamespace $namespaces}}
+
+#Test Throughput Setup //TODO
+{{$SCHEDULER_THROUGHPUT_THRESHOLD := DefaultParam .CL2_SCHEDULER_THROUGHPUT_THRESHOLD 0}}
+{{$DENSITY_TEST_THROUGHPUT := DefaultParam .DENSITY_TEST_THROUGHPUT 100}}
+{{$MIN_SATURATION_PODS_TIMEOUT := 180}}
+{{$saturationDeploymentTimeout := DivideFloat $totalPods $DENSITY_TEST_THROUGHPUT | AddInt $MIN_SATURATION_PODS_TIMEOUT}}
+
+# Methods that don't work
+# APResponsiveness
+# CPUProfile
+# ETCD Metrics
+# Memory Profile
+# MetricsForE2E
+# Scheduling Metrics
+# OOM Tracking
+# InClusterNetworkLatency
+# DNSLookupLatency
+
+name: simple bring up
+namespace:
+  number: {{$namespaces}}
+tuningSets:
+- name: Uniform5qps
+  qpsLoad:
+    qps: 100
+
+steps:
+- name: Starting Measurements
+  measurements:
+  - Identifier: ScaleUpTimer
+    Method: Timer
+    Params:
+      action: start
+      label: scale_up_time
+  - Identifier: PodStartupLatency
+    Method: PodStartupLatency
+    Params:
+      action: start
+      labelSelector: group = nichotr
+      threshold: 1h
+  - Identifier: ResourceUsageSummary
+    Method: ResourceUsageSummary
+    Params:
+      action: start
+  - Identifier: SchedulingThroughput
+    Method: SchedulingThroughput
+    Params:
+      action: start
+
+- name: Creating saturation pods
+  phases:
+    - namespaceRange:
```

Comment:
Let's use a single namespace for all of the pods. I see no reason to incur the additional API server load of creating additional namespaces.

Reply:
+1